pkg/cfn/manager/nodegroup.go

```diff
@@ -0,0 +1,46 @@
+package manager
+
+import (
+	"fmt"
+
+	"github.com/weaveworks/eksctl/pkg/cfn/builder"
+
+	"github.com/kubicorn/kubicorn/pkg/logger"
+)
+
+func (c *StackCollection) makeNodeGroupStackName(sequence int) string {
+	return fmt.Sprintf("eksctl-%s-nodegroup-%d", c.spec.ClusterName, sequence)
+}
+
+func (c *StackCollection) makeNodeGroupParams(sequence int) map[string]string {
+	return map[string]string{
+		builder.ParamClusterName:      c.spec.ClusterName,
+		builder.ParamClusterStackName: c.spec.ClusterStackName,
+		builder.ParamNodeGroupID:      fmt.Sprintf("%d", sequence),
+	}
+}
+
+func (c *StackCollection) CreateInitialNodeGroup(errs chan error) error {
+	return c.CreateNodeGroup(0, errs)
+}
+
+func (c *StackCollection) CreateNodeGroup(seq int, errs chan error) error {
```

Comment:
Currently `0` is harded to be passed in as seq. What are the plans with this? Are we going to allow multiple node groups?

Reply:
Yes. I think numeric IDs make sense, but we can review that when we get to it. At least calling initial one `nodegroup-0` makes sense, but if user may need to give their extra groups more descriptive names, we can probably cater for it.
Right now I fell like numeric ones could work nicely if we want to say "I want an extra group with different types of instances and some custom IAM roles", e.g. without user having to specify a name... but really it's mostly a placeholder for now.